area.go

```diff
@@ -44,7 +45,21 @@ func (area *Area) Update(content string) {
 	SetTarget(area.writer) // Temporary set the target to the Area's writer so we can use the cursor functions
 	area.Clear()
 	SetTarget(oldWriter) // Reset the target to the old writer
-	fmt.Fprintln(area.writer, content)
+	lines := strings.Split(content, "\n")
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>assignments should only be cuddled with other assignments (wsl)